### ISSUE TITLE
No text in main menu on Russian language

### ISSUE THREAD

**Issue by AptemCat** [FAForever/faf-linux#48]
## Issue
I have my game language set as Russian, and I have no text in the main menu of the game (see image 1), and partially in the FAF lobby (see image 2). However, the tooltips still work and the text in English is still present (see image 3). The issue disappears, when the game language is set to English. Also the issue is not present in the Windows version. Unable to confirm the issue on the vanilla Steam version of the game, because it simply does not run with or without the recommended launch parameters.
## Attempted fixes
I've tried installing fonts from `Supreme Commander Forged Alliance/fonts`, tried putting them into `~/.fonts`, into `C:\Windows\Fonts` on FAF's wineprefix and on Steam's wineprefix for SC:FA (`steamapps/compatdata/9420/pfx`). Also tried renaming the `Supreme Commander Forged Alliance/fonts` to something else, as I saw that suggested on ProtonDB.
## Specs
Ubuntu 24.04, system locale `ru-RU`, `language-pack-en` installed
Wine 9.14 from the `winehq-devel` package through their apt source
Official Steam (from a `.deb` file)
## Images
![image1](https://github.com/user-attachments/assets/09bfcc3e-6c00-43fa-b7de-b0308442c6ee)
![image2](https://github.com/user-attachments/assets/8244515c-3333-4c15-9d75-5420b3772852)
![image3](https://github.com/user-attachments/assets/7b023dbc-08f9-4754-800e-ffbd420c1273)

**Comment by AptemCat** [comment]
> Unable to confirm the issue on the vanilla Steam version of the game, because it simply does not run with or without the recommended launch parameters.

The game and Steam were installed on different drives. I moved the game to the default Steam library and it magically started working.

The text issue still persists on the vanilla Steam version. Should I close this issue, as it is not FAF-related, or is anyone able to help?

**Comment by iczero** [comment]
Can you check to see if there are any spaces or non-ASCII in the file path of your other Steam library?

**Comment by AptemCat** [comment]
> Can you check to see if there are any spaces or non-ASCII in the file path of your other Steam library?

I actually just selected the Steam folder from my Win10 installation as the library. It's `/mnt/windows/Program Files (x86)/Steam/`. I believe the spaces are fine (otherwise how would `Supreme Commander Forged Alliance` work as the game folder name), and the brackets are the source of the problem.

**Comment by iczero** [comment]
Is it on NTFS? Not sure if that affects anything though...

**Comment by AptemCat** [comment]
> Is it on NTFS? Not sure if that affects anything though...

Yeah, the previous library was on NTFS. Now the game's on ext4. Don't believe that affects the text issue though. Do you know anything about that?

**Comment by valignatev** [comment]
The game might depend on some system fonts that are installed on Windows but not on Linux. You might try to snitch something like Arial or idk from `C:\Windows\fonts` of your windows system. Just a speculation but I wouldn't be surprised if that's the case

**Comment by AptemCat** [comment]
@valignatev, don't think it would help, as I've tried installing fonts from the game's own `fonts` folder onto the Linux host. It contains Arial and other fonts, some of which definitely aren't present on any normal Windows machine by default.
![image](https://github.com/user-attachments/assets/6f62fd07-576f-404e-844f-113cd4c0e5c3)
Also what I forgot to mention is I tried doing `protontricks 9420 corefonts` and `winetricks corefonts` on the FAF prefix, which should have installed some Windows fonts including Arial. Did not fix the issue.
But that was before moving the game and reinstalling FAF, so I'm doing it once again.

**Comment by valignatev** [comment]
I see, I see. Maybe the game complains about something font-related in one of the log files?

**Comment by AptemCat** [comment]
Launched a co-op game through FAF, saw that (almost) no text was visible in the lobby, closed the game. Here are the only lines that contain "font":
```
info: DISK: AddSearchPath: 'z:\home\%USER%\.local\share\steam\steamapps\common\supreme commander forged alliance\fonts', mounted as '/fonts/'
<...>
warning: Unable to create document path: C
warning: Unable to create document path: C
warning: Unable to create document path: C
info: adding font file /fonts/arial.ttf
info: adding font file /fonts/arialbd.ttf
info: adding font file /fonts/arialbi.ttf
info: adding font file /fonts/ariali.ttf
info: adding font file /fonts/arialn.ttf
info: adding font file /fonts/arialnb.ttf
info: adding font file /fonts/arialnbi.ttf
info: adding font file /fonts/arialni.ttf
info: adding font file /fonts/ariblk.ttf
info: adding font file /fonts/arlrdbd.ttf
info: adding font file /fonts/butterbe.ttf
info: adding font file /fonts/vdub.ttf
info: adding font file /fonts/wintermu.ttf
info: adding font file /fonts/zeroes_3.ttf
```
The log contains no errors, the only suspicious thing are those three cryptic warning messages.
Here's the full log:
[game_23055634.log](https://github.com/user-attachments/files/16516835/game_23055634.log)

**Comment by labi-le** [comment]
idk how it works, but changing directory name fonts (in game directory) to any other and will start displaying the fonts
